config.lib

```diff
@@ -2900,10 +2901,15 @@ detect_iconv() {
 	# Try to find iconv.h, seems to only thing to detect iconv with
 
 	if [ "$with_iconv" = "1" ] || [ "$with_iconv" = "" ] || [ "$with_iconv" = "2" ]; then
-		iconv=`ls -1 /usr/include 2>/dev/null | grep "iconv.h"`
-		if [ -z "$iconv" ]; then
-			iconv=`ls -1 /usr/local/include 2>/dev/null | grep "iconv.h"`
-		fi
+		# Iterate over search paths
+		iconv=""
+		search_paths=`$cxx_host $OSX_SYSROOT $CFLAGS -E - -v </dev/null 2>&1 | $awk '/#include <...> search starts here:/{flag=1;next}/End of search list./{flag=0}flag'`
```

Comment:
You are searching for an English message string without a specific lang. I'd consider a `LC_ALL=C` needed.